src/main/java/net/camotoy/bedrockskinutility/client/mixin/CapeFeatureRendererMixin.java

```diff
@@ -1,14 +1,24 @@
 package net.camotoy.bedrockskinutility.client.mixin;
 
+import net.camotoy.bedrockskinutility.client.SkinManager;
+import net.minecraft.client.player.AbstractClientPlayer;
 import net.minecraft.client.renderer.RenderType;
 import net.minecraft.client.renderer.entity.layers.CapeLayer;
 import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.PlayerModelPart;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Redirect;
 
 @Mixin(CapeLayer.class)
 public class CapeFeatureRendererMixin {
+
+    @Redirect(method = "render(Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;ILnet/minecraft/client/player/AbstractClientPlayer;FFFFFF)V",
+                at = @At(value = "INVOKE", target = "Lnet/minecraft/client/player/AbstractClientPlayer;isModelPartShown(Lnet/minecraft/world/entity/player/PlayerModelPart;)Z"))
+    public boolean isModelPartShown(AbstractClientPlayer instance, PlayerModelPart playerModelPart) {
```

Comment:
Why is this needed?

Reply:
so the cape will show up? Look at the code and after double check it seems like it is needed for the cape to show up.

Reply:
Redirecting this means that Java capes won't show because our cape map won't include official Java player capes. Also, this isModelPartShown redirects to player entity data and not anything in the rendering system. Is there an instance where this was actually needed?

Reply:
> Redirecting this means that Java capes won't show because our cape map won't include official Java player capes. Also, this `isModelPartShown` redirects to player entity data and not anything in the rendering system. Is there an instance where this was actually needed?

Added so now it also check for `isModelPartShown`, it used to check if the client should render a certain player cape, which in `isModelPartShown` it get entity data which is normally sent my by the server as my knowledge. Which is not going to get send on ViaBedrock, and maybe GeyserMC which I don't know. I double check and the cape seems to not showing up without this part of code, if you have any doubt just debug it yourself.

Reply:
Please remove this part of the code. I understand its use, but I don't want to erase the functionality of hiding a cape if Geyser ever decides to utilize it (technically, a server can already hide the cape of a player if it so desires). I have ran this by RK.

Reply:
> Please remove this part of the code. I understand its use, but I don't want to erase the functionality of hiding a cape if Geyser ever decides to utilize it (technically, a server can already hide the cape of a player if it so desires). I have ran this by RK.

Done.

Reply:
I removed it as you requested.